nemo/collections/asr/modules/conv_asr.py

```diff
@@ -351,7 +422,7 @@ class SpeakerDecoder(NeuralModule, Exportable):
         num_classes (int): Number of unique speakers in dataset
         emb_sizes (list) : shapes of intermediate embedding layers (we consider speaker embbeddings from 1st of this layers)
                 Defaults to [1024,1024]
-        pool_mode (str) : Pooling stratergy type. options are 'gram','xvector','superVector'.
+        pool_mode (str) : Pooling stratergy type. options are 'xvector','tap', 'ecapa'
```

Comment:
Are the older types being deprecated? Raise error explicitly for old values then

Reply:
older poolings were added but we didn't support them. They were just there for experiment and internal research purposes when I was first designing SpeakerNet